### PR TITLE
Request review

### PR DESCRIPTION
**Pull request by avichazen**
Hi Laura - please review and approve pull request for merge

### CODE REVIEW

**Comment by lsnovich** [comment]
Your diagram needs a small change
Where you have triangles, please make them rectangles. 

thanks!

**Comment by avichazen** [comment]
@lsnovich workflow updated. please approve merge pull request

**Comment by lsnovich** [comment]
I am not sure the diagram will be visable. something looks odd with the code / filename

**Comment by avichazen** [comment]
@lsnovich I made the diagram in word, made a snippet from the screen, placed it in "issues", and copied that code into the readme.
Do you have a better way to suggest how to insert the workflow?

File: README.md [hunk 1]
```diff
@@ -45,6 +55,12 @@ Dinner|YYYY-MM-DDT17:00:00-05:00 - YYYY-MM-DDT11:59:59-05:00 |17:00 - 23:59
...
+# Resources 
+
+The General Putnam Motel Diner API contains the following resources:
+* [GET Reference Guide](https://github.com/avichazen/api-doc-example-obw/blob/main/References/GET%20Reference%20Guide.md)
+* [POST Reference Guide](https://github.com/avichazen/api-doc-example-obw/blob/main/References/POST%20Reference%20Guide.md)
+
 # Workflow for Front End Developers
 
-![simple workflow](https://user-images.githubusercontent.com/70967800/100592798-42488e80-3300-11eb-8f01-8092296205fd.png)
+![simple workflow](https://user-images.githubusercontent.com/70967800/101278255-d912c080-37c2-11eb-8b52-c46b68bfbc19.png)
```
Comment (lsnovich) [inline README.md]: what is the name of the graphic?
it should be more like
```'simple workflow```

**Review by lsnovich — COMMENTED** [review]
a much better way is detailed in the [FAQ](https://docs.google.com/document/d/1E-1HT-fn1I8un6dUnrNzCA-YopOwl4DFht9VdYlMAy4/edit#)  - I just updated the FAQ with a more detailed description. 
Please take a look

File: README.md (hunk 1 above)
Comment (avichazen) [reply README.md]: Workflow diagram fixed. Please review

File: README.md [hunk 2]
```diff
@@ -16,6 +16,16 @@ Our company has been requested to provide a POC (proof of concept) for the app.
 
 If General Putnam Motel Diner management approves POC, they will pay for the final app.
 
+# Authentication
+
+To make an order, there is an authentication process.
+See here for [Authentication](https://github.com/avichazen/api-doc-example-obw/blob/main/Authentication.md)
```
Comment (lsnovich) [inline README.md]:
```suggestion
Refer to [Authentication](https://github.com/avichazen/api-doc-example-obw/blob/main/Authentication.md), for more information. 
```

File: README.md [hunk 3]
```diff
@@ -16,6 +16,16 @@ Our company has been requested to provide a POC (proof of concept) for the app.
...
+
+# Error Codes
+
+The General Putnam Motel Diner API follows standard HTTP status codes for success or failure of an API call.  
+See here for [Error Codes](https://github.com/avichazen/api-doc-example-obw/blob/main/Error%20Codes.md)
```
Comment (lsnovich) [inline README.md]:
```suggestion
Refer to [Error Codes](https://github.com/avichazen/api-doc-example-obw/blob/main/Error%20Codes.md) for more information.
```

**Review by lsnovich — CHANGES_REQUESTED** [review]
2 small changes and then you can merge

**Comment by avichazen** [comment]
Thanks Laura - merge applied

**Pull request merged**